docs/reference/esql/metadata-fields.asciidoc

```diff
@@ -17,20 +17,24 @@ supported ones are:
   * `_version`: the source document's version. The field is of the type
   <<number,long>>.
 
-To enable the access to these fields, the <<esql-from,`FROM`>> source command needs
-to be provided with a dedicated directive:
+  * <<mapping-source-field,`_source`>>: the original JSON document body 
+  that was passed at index time. The field is loaded as a special `_source` 
```

Comment:
This is _technically_ incorrect in case of `"_source": {"mode": "synthetic"}`; since we don't store the original JSON document, the returned `_source` will generally have re-ordered fields, or timestamps like `2023-01-01` will be expanded to `2023-01-01T00:00:00.000Z`.

Maybe like this:
```suggestion
  that was passed at index time (or a reconstructed version thereof in case synthetic source is enabled). The field is loaded as a special `_source` 
```
with a link to https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-source-field.html#synthetic-source ?

Synthetic source is not yet GA, so I wonder if we need to include it here for completeness' sake or not.

Reply:
Thanks, yes if we want to mention synthetic-source we should have a link

Reply:
Added in https://github.com/elastic/elasticsearch/pull/105237/commits/f53d4bf3645e1bba912d0dd1f7fa1a423f8768cc